cypress/integration/clue/full/teacher_tests/teacher_chat_spec.js

```diff
@@ -0,0 +1,102 @@
+import TeacherDashboard from "../../../../support/elements/clue/TeacherDashboard";
+import ClueCanvas from "../../../../support/elements/clue/cCanvas";
+import ChatPanel from "../../../../support/elements/clue/ChatPanel";
+import ResourcesPanel from "../../../../support/elements/clue/ResourcesPanel";
+/**
+ * Notes:
+ *
+ * Teacher dashboard test needs static data from 'clueteachertest's class 'CLUE'
+ * Here is the ID for the class in firebase: a1f7b8f8b7b1ad1d2d6240c41bd2354d8575ee09ae8bd641
+ *
+ * Currently issues with problem switcher/class switcher. Maybe split these into two tests. Have this test
+ * log into portal with data that doesn't need to be static.
+ *
+ * -> This may also help with issue when verifying read-only student canvases which is currently looping through
+ *    all of the students in the dashboard's current view
+ */
+
+let dashboard = new TeacherDashboard();
```

Comment:
```suggestion
let dashboard = new TeacherDashboard;
```
For consistency

Reply:
Thank you, I've fixed this with my latest commit along with the fixes for the failing test.